src/ErrorHandler.php

```diff
@@ -265,7 +265,7 @@ public function addExceptionHandlerListener(callable $listener): void
      */
     private function handleError(int $level, string $message, string $file, int $line, ?array $errcontext = []): bool
     {
-        if (0 === error_reporting()) {
+        if (!(error_reporting() & $level)) {
```

Comment:
Since we are comparing a bitmask I think it would be clearer to write `0 === (error_reporting() & $level)`

Reply:
Yep, will change it in the evening hours and will also add a CHANGELOG entry.

Reply:
Changed :)